Makefile

```diff
@@ -1,6 +1,6 @@
 TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
 RELEASE_VERSION ?= latest
-CHART_PATH ?= ./helm-charts/strimzi-kafka-operator/
+CHART_PATH ?= ./helm-charts/strimzi-cluster-operator/
```

Comment:
These renamings are wrong. They should not be here. The Helm chart name should be `strimzi-kafka-operator` here.